yarn.lock

```diff
@@ -2811,7 +2811,7 @@
   dependencies:
     defer-to-connect "^1.0.1"
 
-"@toruslabs/torus-embed@^0.2.3":
+"@toruslabs/torus-embed@0.2.3":
```

Comment:
Can this affect Andy?

Reply:
@andrewdamelio 

This happened automatically when I called `yarn` but I can revert it if it's a problem.

Reply:
no need to revert @robert-e-davidson3  👍